docs/source/building/meta-yaml.rst

```diff
@@ -439,6 +440,42 @@ emitted during the build process and the variable will remain
 undefined.
 
 
+.. _pin_downstream:
+
+Pin downstream
```

Comment:
Yeah, with conda's need to be more flexible, it's important to allow different packages to specify their minimum and maximum requirements.  Otherwise we end up with unsatisfiable dependencies very easily.  For you, you are building a single unique system (and moreover, building is less of a concern for you than for consumers of conda), so it's less of an issue.